.github/workflows/daily-build.yml

```
@@ -32,6 +31,7 @@ jobs:
         run: |
           ./gradlew clean build $GRADLE_SKIP_TASKS $BUILD_USING_DOCKER
```

After removing the above step (Set environment variable), we can directly put the Gradle build options here. 

```suggestion
          ./gradlew clean build -P buildUsingDocker=nightly
```